testing/pom.xml

```diff
@@ -1,165 +1,171 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Not sure why the huge difference -- could be different line endings? Just a guess, though. Main thing is to make sure your branch is up-to-date with the upstream repo.